docs(api): Add more details on thermocycler docs (#4029)
Opentrons/opentrons: api/docs/source/v2/new_modules.rst

For the temperature module this will return a string stating whether it's ``'heating'``, ``'cooling'``, ``'holding at target'`` or ``'idle'``.
For the magnetic module this will return a string stating whether it's ``'engaged'`` or ``'disengaged'``.

> For the Thermocycler Module module this will return ``'holding at target'`` or ``'idle'``. There are more detailed status checks which can be found in at :ref:`thermocycler-section`

Reviewer: ```suggestion
For the Thermocycler Module this will return ``'holding at target'`` or ``'idle'``. There are more detailed status checks which can be found in at :ref:`thermocycler-section`
```

    until you decide to deactivate the module from the Opentrons App as described above.

Lid Motor Control
^^^^^^^^^^^^^^^^^
The Thermocycler Module supports temperature control with the lid open and closed. When the lid of the Thermocycler Module is open, the pipettes can access the contained 96-well labware.
You can control the lid with the methods below.

Open Lid
++++++++

.. code-block:: python

    tc_mod.open_lid()

Close Lid
+++++++++

.. code-block:: python

    tc_mod.close_lid()

Lid Temperature Control
^^^^^^^^^^^^^^^^^^^^^^^
As mentioned before, users have access to controlling when a lid temperature is set. It is recommended that you set
the lid temperature before executing a Thermocycler Module profile, described later. The range of the Thermocycler Module lid is
37°C to 110°C.

Set Lid Temperature
+++++++++++++++++++
:py:meth:`.ThermocyclerContext.set_lid_temperature` takes in one parameter which is the temperature you wish the lid to be set to. The protocol will only proceed
once the lid temperature has been reached.

.. code-block:: python

    tc_mod.set_lid_temperature(temperature)

Block Temperature Control
^^^^^^^^^^^^^^^^^^^^^^^^^

> To set the aluminum block temperature inside the Thermocycler Module, you can use the method :py:meth:`.ThermocyclerContext.set_block_temperature`. It takes in three parameters

Reviewer: ```suggestion
To set the aluminum block temperature inside the Thermocycler Module, you can use the method :py:meth:`.ThermocyclerContext.set_block_temperature`. It takes in four parameters
```

.. code-block:: python

>         tc_mod.set_block_temperature(4, hold_time=60, ramp_rate=0.5)

Author: Should be updated, good catch.

minutes with ``hold_time_seconds`` and ``hold_time_minutes``. **Note** This is *only* for controlling the temperature of the `block` in the thermocycler.

.. code-block:: python

        profile = [
          {temperature: 10, hold_time_seconds: 30},
          {temperature: 10, hold_time_seconds: 30},
          {temperature: 10, hold_time_seconds: 30}]

        tc_mod.execute_profile(steps=profile, repetitions=30)

Thermocycler Status
^^^^^^^^^^^^^^^^^^^
Throughout your protocol, you may want particular information on the current status of your thermocycler. Below are
a few methods that allow you to do that.

Lid Position
++++++++++++
Returns the current status of the lid position. It can either be `open`, `closed` or `in-between`.

.. code-block:: python

    tc_mod.lid_position

Lid Temperature Status
++++++++++++++++++++++
Returns the current status of the lid temperature. It can either be `holding at target`, `cooling`, or `heating`.

.. code-block:: python

    tc_mod.lid_temperature_status

Block Temperature Status
++++++++++++++++++++++++

> Returns the current status of the block temperature. It can either be `holding at target`, `cooling`, or `heating`.

Reviewer 2: ```suggestion
Returns the current status of the block temperature. It can be one of the strings ``'holding at target'``, ``'cooling'``, or ``'heating'``.
```

Same here, this should be "block temperature controller" or something

.. code-block:: python

    tc_mod.block_temperature_status

Thermocycler Deactivate
^^^^^^^^^^^^^^^^^^^^^^^

> At some points in your protocol, you may want to deactivate certain aspects of your thermocycler. Below you will find three methods,

Reviewer 2: ```suggestion
At some points in your protocol, you may want to deactivate certain aspects of your thermocycler. This is done with three methods,
```
